src/main/java/core/basesyntax/db/DataBase.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.db;
+
+import core.basesyntax.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataBase {
+
+    public static final List<FruitTransaction> fruitDataBase = new ArrayList<>();
```

Comment:
in DB we should store state of our storage - i.e. how many fruits of each type we have. We do not store the transactions themselves, we process them on the fly